examples/example_workers/chemistry_worker/stubs.py

```diff
@@ -0,0 +1,41 @@
+"""Code generated from chemistry_worker namespace. Please do not edit."""
+
+from typing import NamedTuple, Sequence, Protocol
+from tierkreis.controller.data.models import TKR, OpaqueType
+
+
+class CompleteActiveSpace(Protocol):
+    @property
+    def n(self) -> int: ...  # noqa: F821 # fmt: skip
+    @property
+    def n_ele(self) -> int: ...  # noqa: F821 # fmt: skip
+
+
+class Molecule(Protocol):
+    @property
+    def geometry(self) -> Sequence[tuple[str, Sequence[float]]]: ...  # noqa: F821 # fmt: skip
+    @property
+    def charge(self) -> int: ...  # noqa: F821 # fmt: skip
+    @property
+    def basis(self) -> str: ...  # noqa: F821 # fmt: skip
+
+
+# TODO: Manually added, need to add codegen that does the same.
```

Comment:
Flagging this as it's a bit tricky to do, may need to add a new kind of class potentially